### PR TITLE
fix: Audio notification instance not updating in realtime

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

This PR will fix the dashboard audio notification instance not updating in real time when changing the values.

Fixes https://linear.app/chatwoot/issue/CW-3385/audio-notifications-are-not-working-properly

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Change the audio notification values in the profile section and check whether they update in real time without a refresh.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [x] I have performed a self-review of my code
- [x] New and existing unit tests pass locally with my changes
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/javascript/dashboard/routes/dashboard/settings/profile/NotificationPreferences.vue [hunk 1]
```diff
@@ -242,31 +242,6 @@ export default {
...
-        this.updateUISettings({
-          alert_if_unread_assigned_conversation_exist: e.target.checked,
-        });
-      }
-      this.showAlert(this.$t('PROFILE_SETTINGS.FORM.API.UPDATE_SUCCESS'));
-    },
-    handleAudioToneChange(value) {
-      this.updateUISettings({ notification_tone: value });
-      this.showAlert(this.$t('PROFILE_SETTINGS.FORM.API.UPDATE_SUCCESS'));
-    },
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/settings/profile/NotificationPreferences.vue:269]: Unused methods.

**Comment by muhsin-k** [comment]
@iamsivin Please fix the conflicts.

**Comment by iamsivin** [comment]
> @iamsivin Please fix the conflicts.

@muhsin-k Fixed

**Review by muhsin-k — APPROVED** [review]

**Comment by pranavrajs** [comment]
Closing this favor of the updates @scmmishra is working on.

**Pull request closed without merging**